### PR TITLE
Release notes for 2.2.0 and 2.1.3

### PR DESCRIPTION
**Pull request by nacx**

### CODE REVIEW

File: releasenotes/2.1.3.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.3 is the sethirdcond bugfix release after jclouds 2.1.0 and comes with several bugfixes and performance improvements.
```
Comment (gaul) [inline releasenotes/2.1.3.md]: Typo: sethirdcond.

Comment (demobox) [inline releasenotes/2.1.3.md]:
```suggestion
Apache jclouds 2.1.3 is the third bugfix release after jclouds 2.1.0 and comes with several bugfixes and performance improvements. 
```

File: releasenotes/2.2.0.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.2.0 is a major release after a long period of time. In this release we have focused on modernizing the
```
Comment (demobox) [inline releasenotes/2.2.0.md]:
```suggestion
Apache jclouds 2.2.0 is a first major release after a long period of time. In this release we have focused on modernizing the
```

File: releasenotes/2.2.0.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+codebase and some dependencies to upgrade to Java 8 and prepare to further upgrade to newer Java versions.
```
Comment (demobox) [inline releasenotes/2.2.0.md]:
```suggestion
codebase and some dependencies to upgrade to Java 8 and prepare to upgrade to newer Java versions in future.
```

Comment (demobox) [reply releasenotes/2.2.0.md]: Or "soon"?

**Comment by demobox** [comment]
> nacx merged commit b904af0 into apache:master 16 hours ago

Many thanks for taking care of this, Ignasi!

**Pull request merged**